quickwit-storage/src/local_file_storage.rs

```diff
@@ -46,18 +46,31 @@ impl fmt::Debug for LocalFileStorage {
 }
 
 impl LocalFileStorage {
-    /// Creates a file storage instance given a root path.
-    pub fn new(root: PathBuf) -> Self {
-        Self { root }
-    }
-
     /// Creates a file storage instance given a uri
+    /// Both scheme `file:///{path}` and `file://{path}` are accepted.
```

Comment:
It does not solve #53 contrary to what the comment says.
(it does solve #208 and #126 however)